engines/sci/sound/music.h

```diff
@@ -195,6 +195,7 @@ class SciMusic : public Common::Serializable {
 	void pauseAll(bool pause);
 	bool isAllPaused() const { return (_globalPause > 0); }
 	void resetGlobalPauseCounter();
+	bool isGlobalPauseActive() const;
```

Comment:
This function has the same implementation `isAllPaused()`. Why not use that instead?

Reply:
Didn't see that one. Sure, we should use that...